Makefile

```diff
@@ -15,6 +15,12 @@ dev: generate
 quickdev: generate
 	@TF_QUICKDEV=1 TF_DEV=1 sh -c "'$(CURDIR)/scripts/build.sh'"
 
+# Shorthand for quickly building the core of Terraform. Note that some
+# changes will require a rebuild of everything, in which case the dev
+# target should be used.
+core: generate
```

Comment:
Should this be `core-dev` to match with all of the other uses of "dev" to signify "just build for this machine, and don't cross-compile for other targets"?